specification/resources/databases/patch_database_config.yml

```
@@ -0,0 +1,45 @@
+operationId: patch_database_config
+
+summary: Update the Database Configuration for an Existing Database
+
+description: |
+  To update configuration for an existing database cluster, send a PATCH request to
```

@yGabriel should there be a `the` after `To update`?

```suggestion
  To update the configuration for an existing database cluster, send a PATCH request to
```